superset/superset_rmv.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=C,R,W
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import re
+
+from flask_appbuilder.security.views import RoleModelView
+from flask_babel import lazy_gettext as _
+
+
+class SupersetRoleModelView(RoleModelView):
```

Comment:
Let's update the file name to remove the abbreviation (`superset_role_model_view.py` rather than `superset_rmv.py`. Clarity > Brevity.